sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -381,6 +381,9 @@ def stage_job_resources(
         if setup_options.sdk_location == 'default':
           raise RuntimeError('Cannot find default Dataflow SDK tar file "%s"',
                              sdk_path)
+        elif not setup_options.sdk_location:
+          logging.info('Dataflow SDK will not be staged since '
```

Comment:
Thanks, done. Also changed dropped Dataflow in other references in the file for consistency.